client/src/components/modals_and_popovers/AriaModal.js

```diff
@@ -0,0 +1,57 @@
+import classNames from "classnames";
+import React from "react";
+import { Modal } from "react-bootstrap";
+import FocusLock from "react-focus-lock";
+
+import "./bootstrap_modal_exstension.scss";
+
+export class AriaModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    if (!props.titleId) {
+      throw new Error("Must have a prop titleId");
+    }
+  }
+
+  //takes a string value
+  hide_application_node_aria = (value) => {
+    const { getApplicationNode } = this.props;
+
+    const applicationNode = getApplicationNode();
+    applicationNode.setAttribute("aria-hidden", value);
+  };
+
+  componentDidMount() {
+    this.hide_application_node_aria("true");
+  }
+
+  componentWillUnmount() {
+    this.hide_application_node_aria("false");
+  }
```

Comment:
Looks like the `Modal` component has the `aria-modal="true"` attribute which should make this unnecessary (although it's relatively new for screen readers, so it's potentially worth keeping this step for backwards compatibility).

I don't see `getApplicationNode` actually being used though, so if this is some leftover code from elsewhere then feel free to drop it and rely on the `react-bootstrap` `Modal` handling things correctly.

Reply:
Oh, double checked, the `Modal` does already take care of also setting `aria-hidden` everywhere outside of the open modal, so that's good!